master/buildbot/master.py

```diff
@@ -139,6 +139,9 @@ def __init__(self, basedir, configFileName=None, umask=None, reactor=None, confi
         self.name = self.name.decode('ascii', 'replace')
         self.masterid = None
 
+    def stopReactor(self):
+        self.reactor.stop()
```

Comment:
patching reactor.stop is not enough anymore with recent twisted, as it is repatched in _wait(), the function, which waits for the test's deferred.